style.css

```diff
@@ -0,0 +1,1086 @@
+* {
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+}
+
+/* from Carlos */
+
+.mobile-nav {
+  display: none;
+}
+
+nav a {
+  text-decoration: none;
+  color: #555;
+  font-family: Maquette, Helvetica Neue, Arial, sans-serif;
+}
+
+.top-nav {
+  width: 100%;
+  display: inline-flex;
+  background-color: #131313;
+  justify-content: space-between;
+  height: 30px;
+  align-items: center;
+}
+
+.topNav-left {
+  display: flex;
+  align-self: flex-end;
+  height: 100%;
+  align-items: center;
+  justify-content: space-between;
+  width: 30%;
+  margin-left: 20px;
+}
+
+.topNav-right {
+  display: flex;
+  align-self: flex-end;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```CSS
.flex{
  display: flex;
}
```
**Html file**

```HTML
<div class="flex">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_